pkg/services/folder/folderimpl/sqlstore_test.go

```diff
@@ -391,11 +392,7 @@ func TestIntegrationGet(t *testing.T) {
 		UID:         util.GenerateShortUID(),
 		ParentUID:   f.UID,
 	})
-
-	t.Cleanup(func() {
-		err := folderStore.Delete(context.Background(), []string{f.UID}, orgID)
-		require.NoError(t, err)
-	})
```

Comment:
[question] [non-blocker] Why is this no longer needed?

Reply:
is not required as long as we keep [truncating the test database](https://github.com/grafana/grafana/blob/40a08a7720e65c3c6fcd03d437d78b5bc9d7b8cb/pkg/services/sqlstore/sqlstore.go#L543) on each test
also, it did delete only one of the table that they were created during the test